cloudinit/config/cc_users_groups.py

```diff
@@ -78,6 +78,12 @@
     If specifying a sudo rule for a user, ensure that the syntax for the rule
     is valid, as it is not checked by cloud-init.
 
+.. note::
+    Following configurations from the above list are applicable on already
+    existing user(s).
+    - 'plain_text_passwd', 'hashed_passwd', 'lock_passwd', 'sudo',
+      'ssh_authorized_keys', 'ssh_redirect_user', 'cloud_public_ssh_keys'.
```

Comment:
`cloud_public_ssh_keys` is not a user-facing configuration option; it is added to the user-supplied configuration internal to cloud-init: https://github.com/canonical/cloud-init/blob/6ac71fdf22db43269cf9b068517290b937ec31b5/cloudinit/config/cc_users_groups.py#L139 https://github.com/canonical/cloud-init/blob/6ac71fdf22db43269cf9b068517290b937ec31b5/cloudinit/config/cc_users_groups.py#L164

We shouldn't include it in this list:
```suggestion
      'ssh_authorized_keys', 'ssh_redirect_user'.
```